packages/gestalt/src/TableHeaderCell.js

```diff
@@ -4,28 +4,49 @@ import cx from 'classnames';
 import styles from './Table.css';
 
 type Props = {|
+  /**
+   * The content of the table cell.
+   */
   children: Node,
+  /**
+   * Private prop required for sticky columns
+   */
   shouldBeSticky?: boolean,
+  /**
+   * Private prop required for sticky columns
+   */
   shouldHaveShadow?: boolean,
+  /**
+   * `colSpan` defines the number of columns a cell should span.
+   */
   colSpan?: number,
+  /**
+   * `rowSpan` defines the number of rows a cell should span.
+   */
   rowSpan?: number,
+  /**
+   * The scope attribute specifies whether a header cell is a header for a column, row, or group of columns or rows. The scope attribute has no visual effect in ordinary web browsers, but can be used by screen readers.
+   */
   scope?: 'col' | 'colgroup' | 'row' | 'rowgroup',
+  /**
+   * Private prop required for sticky columns
+   */
   previousTotalWidth?: number,
 |};
 
 /**
- * https://gestalt.pinterest.systems/table
+ * Subcomponent of [Table](https://gestalt.pinterest.systems/table).
+ * Use [Table.HeaderCell](https://gestalt.pinterest.systems/table#Table.HeaderCellProps) to define a header cell in Table.
  */
-export default function TableHeaderCell(props: Props): Node {
-  const {
-    children,
-    colSpan,
-    scope,
-    rowSpan,
-    shouldBeSticky,
-    previousTotalWidth,
-    shouldHaveShadow,
-  } = props;
+export default function TableHeaderCell({
+  children,
```

Comment:
nit: do you mind alphabetizing these while you're here?

Reply:
dONE